openapi-specs/schema.json

```diff
@@ -149,7 +149,8 @@
               "default": "match",
               "description": "set the scope for the manage role permission"
             }
-          }
+          },
+          "required": ["scope"]
```

Comment:
if we have a default value, do we also need to set it required? I think my expectation from the client would be to not send anything if the users chooses nothing explicitly and just let the server set its default